test/Storage/MessageTest.php

```diff
@@ -39,41 +39,19 @@ public function testInvalidFile()
         $this->fail('no exception raised while loading unknown file');
     }
 
-    public function testIsMultipart()
-    {
-        $message = new Message(['file' => $this->_file]);
-
-        $this->assertTrue($message->isMultipart());
-    }
-
-    public function testGetHeader()
-    {
-        $message = new Message(['file' => $this->_file]);
-
-        $this->assertEquals($message->subject, 'multipart');
-    }
-
-    public function testGetDecodedHeader()
-    {
-        $message = new Message(['file' => $this->_file]);
-
-        $this->assertEquals('Peter Müller <peter-mueller@example.com>', $message->from);
-    }
-
-    public function testGetHeaderAsArray()
```

Comment:
Having these as separate tests ensures that we can have granular tests for each behavior. Pushing all assertions into a single method makes identifying breakages harder, as the first assertion that fails causes any following assertions to never execute, which means we often end up playing whack-a-mole (fix one assertion, only to find the same test fails on an additional assertion). Please re-instate these discrete methods.